### PR TITLE
Small updates

### PR DESCRIPTION
**Pull request by aju**
closes https://github.com/brainly/style-guide/issues/455
closes https://github.com/brainly/style-guide/issues/459
closes https://github.com/brainly/style-guide/issues/674
closes https://github.com/brainly/style-guide/issues/610

### CODE REVIEW

File: src/sass/_config.scss [hunk 1]
```diff
@@ -66,6 +66,7 @@ $fontSizes: (
   xxlarge: 5rem,      // 80px
   obscure: 0.75rem,   // 12px
   standout: 1.125rem, // 18px
+  subheadline: 1.25rem, // 20px
```
Comment (vergilius) [inline src/sass/_config.scss]: where it is used?

Comment (aju) [inline src/sass/_config.scss]: Its used in header primary xsmall - it needs to be added yet

**Comment by kdzwinel** [comment]
expose all options to the docs!!

**Comment by aju** [comment]
screen shot 2016-03-17 at 11 39 32
screen shot 2016-03-17 at 11 39 38

**Comment by aju** [comment]
@kdzwinel @vergilius updated

**Comment by vergilius** [comment]
:+1:

**Comment by matzimowski** [comment]
sexi!
:+1:

**Comment by kdzwinel** [comment]
ok

**Pull request merged**